src/test/kotlin/minesweeper/domain/MineSweeperTest.kt

```diff
@@ -0,0 +1,133 @@
+package minesweeper.domain
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class MineSweeperTest : StringSpec({
+
+    "주변 셀에 지뢰가 없으면 0 으로 표시한다." {
+        val mineMap = MineMap(2, 2, 0)
+        val sweptMineMap = MineSweeper.sweep(mineMap.map())
+
+        sweptMineMap.flatten().count { it is MineCell } shouldBe 0
+        sweptMineMap.flatten().all { it.mineCountAround == 0 } shouldBe true
+    }
+
+    "기준 셀 오른쪽에 지뢰가 있으면 1로 표시된다." {
+        val sweptMineMap = MineSweeper.sweep(listOf(listOf(Cell(), MineCell())))
```

Comment:
"기준 셀 왼쪽 위에 지뢰가 있으면 1로 표시된다." 이하 테스트처럼 두개 경우도 시각적으로 표현되면 좋겠습니다.
```suggestion
        val sweptMineMap = MineSweeper.sweep(
            listOf(
                listOf(Cell(), MineCell())
            )
        )
```
dsl 을 활용한다면 좀 더 테스트 가독성을 올릴수 있을것 같습니다. dsl 연습차원에서 여유가 되신다면 도전해 보시는것도 좋을것 같아요.